src/components/Offer/Offer.tsx

```diff
@@ -7,185 +7,202 @@ import { ImageWithinOffer } from '../ImageWithinOffer/ImageWithinOffer'
 import { PriceOfOffer } from '../PriceOfOffer/PriceOfOffer'
 import { Rating } from '../rating/Rating'
 import { SellerPreviewInfo } from '../SellerPreviewInfo/SellerPreviewInfo'
+import { mediaQueriesPx } from '../../mediaQueries'
+export interface OfferProps {
+	imageForOffer: string
+	imageDescription: string
+	offerName: string
+	currency: string
+	amount: number
+	unit: string
+	star: number
+	ratingDescription: string
+	imageForRating?: string
+	amountOfProduct: number
+	offerDescription: string
+	isLearnEnabled: boolean
+	detailButtonText: string
+	favoriteButtonText: string
+	isFavourited: boolean
+	profile: string
+	name: {
+		firstName: string
+		lastName: string
+	}
+	location: {
+		country: string
+		city: string
+	}
+}
+
+export const Offer: React.FC<OfferProps> = ({
+	imageForOffer,
+	imageDescription,
+	offerName,
+	currency,
+	amount,
+	unit,
+	star,
+	ratingDescription,
+	amountOfProduct,
+	offerDescription,
+	isLearnEnabled,
+	detailButtonText,
+	favoriteButtonText,
+	isFavourited,
+	profile,
+	name,
+	location,
```

Comment:
I would prefer if offer would be an object with defined interface that is also used in offerSlice. This way we avoid retyping each prop